packages/datadog-instrumentations/src/oracledb.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
+
+const {
+  channel,
+  addHook,
+  AsyncResource
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const connectionAttributes = new WeakMap()
+const poolAttributes = new WeakMap()
+
+const executeStartChannel = channel('apm:oracledb:execute:start')
+const executeErrorChannel = channel('apm:oracledb:execute:error')
+const executeEndChannel = channel('apm:oracledb:execute:end')
+
+function finish (err) {
+  if (err) {
+    executeErrorChannel.publish(err)
+  }
+  executeEndChannel.publish(undefined)
+}
+
+addHook({ name: 'oracledb', versions: ['5'] }, oracledb => {
+  shimmer.wrap(oracledb.Connection.prototype, 'execute', execute => {
+    return function wrappedExecute (dbQuery, ...args) {
+      if (!executeStartChannel.hasSubscribers) {
+        return execute.apply(this, arguments)
+      }
+
+      if (arguments.length && typeof arguments[arguments.length - 1] === 'function') {
+        const cb = arguments[arguments.length - 1]
+        const outerAr = new AsyncResource('oracledb:outer-scope')
```

Comment:
This should be named in a way that would either not conflict or use the default.

Reply:
There is no default (unless using the static `bind()`).

Conflicts don't actually matter here, AFAIK. We're already putting `bound-anonymous-fn` in a lot of other places that conflict with each other. This is just to give it some indication of what it is when using a debugging tool like `async-break-finder`.

Reply:
Would prefixing with `apm` suffice here?